apps/mobile/src/slices/auth/actions.ts

```diff
@@ -1,21 +1,37 @@
 import { createAsyncThunk } from "@reduxjs/toolkit";
 
+import { storage, StorageKey } from "~/libs/packages/storage/storage";
 import { type AsyncThunkConfig } from "~/libs/types/types";
 import {
+	type UserDto,
 	type UserSignUpRequestDto,
 	type UserSignUpResponseDto,
 } from "~/packages/users/users";
 
 import { name as sliceName } from "./auth.slice";
 
+const getAuthenticatedUser = createAsyncThunk<
+	UserDto,
+	undefined,
+	AsyncThunkConfig
+>(`${sliceName}/get-authenticated-user`, async (_, { extra }) => {
+	const { authApi } = extra;
+	const response: UserDto = await authApi.getAuthenticatedUser();
```

Comment:
```suggestion
	const user = await authApi.getAuthenticatedUser();
```
Do we need it? I think the type comes from authApi

Reply:
Update